README.md

```diff
@@ -58,16 +59,33 @@ const blogOptions: BlogOptions = {
     Contact: "/contact",
     Projects: "/projects",
   },
+  lang: "en",
 };
 ```
 
 There's also an option to configure the `postsPerPage` which defaults to 10.
 
+You can create custom Localization and add it as the second parameter for blogPlugin().
+
+```ts
+const blogLocalization: Localization = {
+  attribution: "By",
+  nextPage: "Next Page →",
+  previousPage: "← Previous Page",
+  nextPost: "Next Post →",
+  previousPost: "← Previous Post",
+  continueReading: "Continue reading →",
+  noPostsFound: "No posts found. Start writing!",
+  blogTitleEnding: " — Blog",
+  archiveTitleEnding: " — Archive",
+  authorTitleEnding: " — Author Archive",
```

Comment:
I'm open to suggestions on how to improve this default as well. But I guess it's fine for now... (And yes, I'm aware that these are my original defaults, I just was never fully satisfied with them!)

Reply:
I think this is good for now. Its straight to the point and users can always override them if they want.